python/pycryptodome/security/insecure-cipher-algorithm-rc2.py

```diff
@@ -19,9 +19,11 @@
 iv = Random.new().read(pycrypto_arc2.block_size)
 # ruleid:insecure-cipher-algorithm-rc2
 cipher = pycrypto_arc2.new(key, pycrypto_arc2.MODE_CFB, iv)
+# deepruleid:insecure-cipher-algorithm-rc2
 msg = iv + cipher.encrypt(b'Attack at dawn')
```

Comment:
Why `deepruleid:`? symbolic propagation propagates the expression `pycrypto_arc2.new(...)` to cipher, resulting in two results: one on the def of cipher and the other is on its use. This should be the case for all of the pycryptodome rules

Reply:
Similar as above